common/constants.js

```diff
@@ -31,6 +31,12 @@ module.exports = Object.freeze({
   },
   EVENTMESH_POLLER_DELAY: 200,
   UNLOCK_RESOURCE_POLLER_INTERVAL: 3000,
+  BACKUP_OPERATION: {
```

Comment:
one feedback , ideally it would be good to use the same constants everywhere i.e in normal code and tests .. i see that the test's have hard coded values.

Reply:
Used constant